hypothesis-python/docs/database.rst

```diff
@@ -20,20 +20,6 @@ workflow considerably by making sure that the examples you've just found are rep
 The database also records examples that exercise less-used parts of your
 code, so the database may update even when no failing examples were found.
 
---------------
-File locations
---------------
```

Comment:
I feel like removing this entirely as a top level piece of documentation makes it harder to understand Hypothesis's default behaviour. I'm not quite sure what needs to go here in its place, but I think something to the tune of "By default Hypothesis uses a `DirectoryBasedExampleDatabase` (linked) which etc" here would be helpful.

Reply:
That's just moved [one paragraph down](https://hypothesis--2622.org.readthedocs.build/en/2622/database.html#exampledatabase-implementations) to the first sentence under "Implementations".  Happy to move that if you have a preferred location?

(also can't merge until you hit `approve`, so you'll need to do at least the null review)

Reply:
So it is. Sorry, apparently I had reading comprehension fail. Glad @kxepal did a more thorough review than me and approved it!